internal/shardpack/shardpack.go

```
@@ -0,0 +1,361 @@
+			return noop, err
+		}
+	}
+	if len(plan.Shards) > 1 {
+		if err := w.deps.WriteFile(filepath.Join(tmp, "cross-shard.md"), []byte(crossShardPack(plan, header)), 0o644); err != nil {
+			return noop, err
+		}
+	}
+	// planFile is a closed struct of strings, ints and slices, so marshalling it
+	// cannot fail; the error is deliberately discarded rather than left as an
+	// unreachable branch.
+	data, _ := json.MarshalIndent(planJSON(plan, header), "", "  ")
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.MarshalIndent` is not checked

(errchkjson)